platform/ios/src/MGLMapView.mm

```diff
@@ -1298,9 +1296,8 @@ - (void)handlePinchGesture:(UIPinchGestureRecognizer *)pinch
     else if (pinch.state == UIGestureRecognizerStateChanged)
     {
         CGFloat newScale = self.scale * pinch.scale;
-        double zoom = log2(newScale);
-        if (zoom < _mbglMap->getMinZoom()) return;
-        
+        double zoom = MAX(log2(newScale), _mbglMap->getMinZoom());
```

Comment:
That is true and, since we’re not otherwise clamping the pinch gesture, I’ve removed this limit in ea81e04924463e86b765f5a76a2d2d7b66ef1521.